addon/loaders/utilities.js

```diff
@@ -37,3 +37,10 @@ export function nodeLoader(loader) {
     return () => RSVP.resolve();
   }
 }
+
+export function uriIncluded (uri, document) {
+  if (document.scripts) {
```

Comment:
When would `document` be present but not `document.scripts`?

Support matrix on https://developer.mozilla.org/en-US/docs/Web/API/Document/scripts suggests it’s been supported for a _very_ long time.